### PR TITLE
Update pt_BR.lang

### PR DESCRIPTION
**Pull request by ArukaAlter**

### CODE REVIEW

File: resources/assets/tconstruct/lang/pt_BR.lang [hunk 1]
```diff
@@ -396,12 +399,18 @@ item.tconstruct.arrow_head.name=Ponta de Flecha
...
 
+item.tconstruct.bolt_core.name=Haste de dardo
+
 # Material specific tools and toolparts
 # item.tconstruct.<tool>.<material>
 # item.tconstruct.pickaxe.wood=Wooden Pickaxe
 
 # Gadgets
 item.tconstruct.slime_boots.name=Botas de Slime
+item.tconstruct.slime_boots.blue.name=item.tconstruct.slime_boots.green.name
```
Comment (bonii-xx) [inline resources/assets/tconstruct/lang/pt_BR.lang]: You don't have the green boots anywhere though

**Comment by ArukaAlter** [comment]
But the name is the same across them (except for the blood boots), there's no color differentiation by name.

**Comment by KnightMiner** [comment]
Yes, but you never actually define the green string, you define the one without a color (now unused). No color in the name is mainly to match slimeballs anyways.

Though it's worth mentioning slimeboots use vanilla localization rather than Tinkers, so setting it to another name is probably not going to localize it again.

**Comment by ArukaAlter** [comment]
It will be at least 15 hours from now for me to be available to fix this properly.

**Pull request closed without merging**